cg/apps/demultiplex/notes.md

```diff
@@ -0,0 +1,100 @@
+We loop over all flowcell directories and check if sequencing and transfering is done. This is indicated by the two
+files RTAComplete.txt and CopyComplete.txt.
+We also create file (demuxcomplete.txt) that indicates if is finished.
+
+## Output to
+
+`/home/proj/stage/demultiplexed-runs/`
+
+## WHY DO WE NEED TO MAKE A DIFFERENCE BETWEEN FLUFFY AND WGS
+
+    - we don't want to add dummy samples to fluffy sample sheet
+    - We don't want to pad the fluffy indexes
+
+
+Distinction can be made from that the 
+    
+    - index length for fluffy is 8 and wgs 10
+    - read length for fluffy is 51 and 151 for wgs
+
+### THIS IS THE PART I DON'T UNDERSTAND ATM
+
+- I think that the file delivery.txt indicates that demultiplexing has started.
```

Comment:
delivery.txt is an artifact from the days of yore. It indicates that the flowcell has finished demultiplexing on thalamus and that the demux dir or out_dir has been 'delivered' to hasta, meaning that the flowcell has been synced to demultiplexed-runs on hasta